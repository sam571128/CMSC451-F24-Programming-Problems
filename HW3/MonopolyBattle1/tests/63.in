42 1764 1000000000
8 18
15 29
38 37
20 42
3 36
23 10
17 29
16 12
6 39
39 16
4 7
9 16
20 37
40 6
32 23
10 20
10 27
27 25
20 17
36 42
30 34
11 38
1 9
42 42
6 6
33 19
19 32
42 40
29 2
20 19
16 24
28 39
38 2
24 5
36 36
7 12
38 3
28 26
37 12
37 6
13 1
28 16
30 12
35 34
42 39
39 19
30 5
34 12
16 27
35 16
27 15
24 1
33 29
41 16
33 15
16 40
32 2
28 14
21 32
41 34
39 39
27 35
24 17
17 35
4 17
9 3
15 8
7 22
6 27
18 19
22 21
14 10
10 2
40 8
34 22
38 17
3 14
11 22
39 9
6 13
2 14
40 28
33 41
29 23
10 8
15 13
28 40
26 36
21 39
38 29
26 31
40 7
16 21
13 25
30 32
25 19
21 29
21 10
27 20
29 40
33 20
23 15
38 18
8 11
4 2
15 15
12 14
37 28
22 11
29 26
8 42
40 32
40 41
6 21
22 40
37 36
24 7
35 25
30 37
33 42
5 40
13 22
15 12
15 18
4 9
33 18
40 22
3 18
10 18
39 31
35 8
32 6
41 21
11 3
28 17
30 15
19 5
27 14
4 10
21 4
15 5
7 26
23 40
13 42
3 19
4 12
23 30
25 6
9 38
27 8
11 18
19 16
13 33
40 11
37 7
21 38
40 14
3 20
42 2
32 25
12 9
26 9
14 18
3 9
30 29
42 35
39 5
23 14
5 28
2 3
7 28
23 9
6 2
15 3
35 12
28 37
41 30
32 1
5 19
39 34
42 8
5 29
39 40
9 1
23 38
37 24
38 11
12 33
2 17
17 21
25 29
34 32
35 24
19 18
37 16
24 28
28 25
40 18
2 23
14 11
3 42
38 10
4 3
13 5
24 22
30 30
40 34
36 26
28 12
7 5
41 11
9 15
37 40
15 7
36 4
4 23
15 39
11 19
23 24
1 34
28 5
15 1
24 9
14 7
29 38
7 41
5 16
37 33
9 22
34 31
2 12
11 14
25 21
38 14
6 1
41 42
40 19
39 3
23 21
34 35
29 6
9 37
15 40
19 24
27 11
22 23
10 13
41 41
8 28
8 22
12 36
3 3
2 37
1 13
12 28
35 1
22 33
13 17
34 18
35 37
5 1
29 17
31 26
16 25
11 40
40 12
17 28
38 32
25 37
12 42
1 23
21 22
14 26
33 3
33 7
27 29
32 11
29 3
18 17
23 12
3 30
14 5
42 37
20 18
37 18
22 38
28 15
41 5
4 6
24 13
5 35
6 22
33 25
36 38
15 23
30 21
22 2
9 18
10 40
14 31
24 23
33 6
1 35
16 9
5 25
10 1
13 4
34 1
19 34
6 23
21 33
31 7
14 38
6 37
9 13
1 5
21 8
8 19
26 35
11 17
30 26
23 1
37 27
21 7
3 28
34 39
31 24
38 26
8 31
4 26
31 18
39 10
38 6
24 12
14 41
33 22
39 15
7 9
33 17
30 39
5 6
26 23
4 20
31 10
6 10
42 38
12 18
2 6
40 1
28 1
33 30
8 32
21 12
16 7
16 33
19 31
21 11
14 35
33 4
27 36
17 20
38 13
27 26
2 16
18 32
26 40
29 42
34 15
33 12
30 41
41 13
30 20
6 38
28 22
39 4
36 3
27 4
35 26
28 23
5 2
22 42
13 12
36 25
1 17
1 8
27 27
35 4
21 36
36 17
13 26
38 31
27 2
22 24
17 11
30 24
23 42
42 15
13 37
35 23
12 29
18 40
2 21
22 7
32 33
10 35
3 38
36 11
6 8
39 23
40 25
10 15
32 15
10 29
29 34
28 18
22 18
5 13
24 37
31 33
14 4
34 42
20 8
13 7
11 21
6 28
1 39
4 28
2 28
15 32
32 14
3 33
41 25
11 39
21 34
38 40
12 21
20 3
42 3
16 1
29 18
34 19
14 25
39 22
12 5
3 25
36 28
35 40
17 12
42 41
25 14
5 3
38 16
7 40
38 33
29 28
13 32
5 36
26 17
23 6
28 3
27 38
34 36
35 18
34 40
41 36
37 25
41 2
31 19
25 28
5 37
11 12
1 14
10 25
21 3
24 26
22 4
40 5
4 16
23 16
36 2
31 21
19 17
18 27
10 28
32 27
34 7
20 1
7 25
35 32
35 13
8 14
11 1
24 8
36 24
4 4
32 13
38 42
39 38
10 17
23 28
21 30
1 33
38 25
26 10
33 13
15 41
14 39
23 18
28 10
21 27
28 31
39 1
17 2
21 6
11 20
39 14
27 22
18 37
22 27
25 17
14 40
18 42
35 35
21 35
11 15
28 27
10 26
23 41
33 26
12 1
35 19
17 22
8 17
6 18
36 41
28 30
22 3
18 14
39 28
29 11
34 17
24 11
38 30
41 20
35 27
31 16
41 29
12 17
6 25
40 24
15 10
37 5
32 19
15 24
9 6
14 2
12 26
24 2
22 20
8 9
16 15
34 20
36 32
42 24
7 23
34 16
33 40
42 23
40 36
7 2
3 39
5 12
3 2
33 24
2 24
17 34
2 11
28 4
15 35
12 6
13 35
11 24
20 36
11 28
25 22
29 13
2 13
39 24
39 2
29 32
35 22
42 16
22 22
22 17
20 32
22 16
9 14
34 29
9 8
11 31
21 41
32 42
42 25
13 24
35 38
32 8
8 20
19 2
39 13
33 16
26 4
22 36
30 31
19 6
34 41
15 25
34 11
25 23
41 14
10 39
42 14
3 32
32 31
34 4
7 8
40 23
2 36
4 14
7 13
2 39
12 35
38 23
42 31
2 34
22 30
42 34
1 41
35 6
24 39
3 26
32 26
4 32
14 24
11 32
25 20
16 41
22 37
36 21
15 19
5 33
39 11
25 42
33 39
13 15
16 13
28 41
20 35
32 32
16 28
5 32
14 9
13 20
40 30
35 28
2 8
8 1
1 19
21 15
9 9
24 14
17 14
7 27
3 15
11 34
35 30
24 4
39 18
2 25
18 13
7 32
37 15
18 12
13 21
13 40
15 9
41 19
7 42
17 32
26 13
24 18
6 33
22 14
3 22
31 14
16 26
26 3
37 35
30 18
9 25
32 5
34 10
37 21
35 21
6 36
30 16
25 35
2 26
17 15
1 31
2 40
31 17
33 5
27 39
6 42
1 6
12 37
30 28
7 3
33 11
18 10
16 22
37 32
35 10
42 33
15 6
23 33
6 41
34 25
23 22
32 29
38 12
14 20
20 23
31 37
5 39
8 21
1 18
13 19
18 25
36 19
18 2
39 35
11 16
3 40
33 36
5 7
12 39
32 17
41 39
22 12
42 32
27 3
36 7
21 42
8 12
14 27
16 11
5 26
4 34
19 39
40 13
38 9
8 41
30 6
36 6
19 42
25 5
20 2
10 38
18 21
8 3
28 21
29 4
26 20
3 11
35 2
23 32
27 1
25 33
38 38
6 17
25 12
8 23
19 13
4 30
41 10
18 15
12 23
37 22
17 13
30 1
26 1
20 27
3 41
37 3
38 28
31 27
6 11
6 30
34 23
17 33
24 33
21 13
40 33
26 30
38 36
34 6
37 14
25 25
9 10
42 18
19 26
5 9
28 35
3 4
34 14
38 15
37 1
25 3
23 3
28 28
22 41
19 35
5 31
4 39
7 21
42 20
27 16
37 38
4 38
35 3
22 9
2 2
6 4
34 9
4 1
9 30
12 34
9 42
22 28
16 30
1 38
3 1
16 10
29 7
33 2
12 12
14 22
15 38
5 4
12 41
36 22
40 10
36 31
18 3
39 20
6 32
10 23
40 39
30 3
26 34
35 33
40 9
21 2
4 42
32 7
6 19
25 8
27 37
17 26
40 21
30 14
39 30
18 24
18 31
19 1
31 20
10 22
10 12
14 17
27 19
10 7
21 19
42 22
40 38
33 35
26 28
36 27
14 1
6 3
24 3
20 10
28 34
36 29
25 10
39 8
10 33
31 28
2 41
19 9
18 9
19 20
4 27
25 18
33 27
24 41
40 4
39 37
19 14
41 22
22 19
16 39
36 14
19 30
16 35
35 5
33 37
3 10
32 3
34 28
10 42
20 11
13 34
26 42
34 30
16 18
14 12
36 30
17 7
16 5
7 18
29 30
14 29
36 13
26 26
4 15
11 5
7 24
10 32
22 26
24 29
1 7
20 12
4 41
27 30
14 14
33 21
11 27
15 17
9 20
19 4
4 25
23 39
19 8
42 19
42 36
5 23
7 33
13 28
18 36
29 22
31 9
26 25
35 31
12 16
30 17
25 40
10 41
29 10
32 37
4 8
37 34
19 11
36 35
40 15
3 13
9 26
18 5
40 16
3 23
32 12
29 16
6 34
22 13
24 38
28 2
2 18
16 4
36 18
23 4
2 33
17 19
17 5
20 26
25 24
11 4
23 37
10 9
19 21
30 13
18 11
6 9
2 9
37 23
29 1
7 6
31 36
25 13
31 1
32 9
25 16
4 33
28 36
21 25
17 41
4 11
16 36
20 24
30 9
36 9
29 36
31 40
28 8
26 18
8 15
40 35
19 23
4 40
34 2
37 31
41 40
22 35
40 2
22 10
39 7
4 21
13 11
36 12
13 29
16 19
8 8
9 17
25 27
39 33
17 4
40 29
18 20
16 8
29 15
32 18
25 1
27 28
23 31
15 26
11 7
31 29
11 35
18 8
36 34
23 29
15 20
40 20
30 40
2 5
14 36
9 21
38 34
22 39
34 26
32 35
3 12
14 33
13 13
10 36
25 31
39 27
12 13
11 37
8 36
11 13
40 26
30 35
4 29
22 29
23 11
30 7
7 20
34 34
29 20
1 37
24 6
8 5
32 10
12 24
9 34
1 30
37 20
4 13
35 15
25 26
39 21
11 41
12 32
34 24
20 41
32 40
2 38
12 20
21 40
31 5
33 1
27 10
31 15
1 28
9 29
3 8
21 16
2 32
9 7
39 12
7 34
1 15
23 27
12 15
6 15
41 32
26 37
10 24
15 31
7 36
23 8
37 42
36 1
8 25
30 4
7 1
22 6
36 20
27 23
30 33
31 34
11 11
14 3
29 33
14 6
22 15
10 37
21 23
20 14
11 26
30 25
1 32
13 8
4 35
37 11
9 33
5 27
14 15
16 6
31 8
3 35
13 6
29 24
26 21
42 29
36 5
41 38
18 28
9 40
26 16
7 39
2 7
19 19
15 33
34 8
29 29
9 31
5 14
19 37
14 8
22 8
7 16
5 24
14 21
31 22
32 38
13 41
42 28
21 37
17 31
22 34
19 36
35 14
11 36
26 19
16 17
26 41
37 37
13 3
33 9
17 42
1 4
41 18
8 4
24 20
28 6
8 37
40 42
13 38
11 30
22 5
24 19
10 6
35 20
7 15
18 38
24 34
2 1
39 41
24 36
18 23
29 41
9 11
28 20
25 4
17 10
8 30
37 41
3 21
37 17
6 35
15 16
19 40
26 6
37 39
20 20
41 35
21 26
24 24
11 2
15 2
37 13
32 41
12 38
5 10
35 29
1 10
41 8
9 41
16 16
31 42
25 30
30 27
11 8
2 15
20 28
30 38
28 32
14 37
36 10
40 40
26 29
35 41
32 30
2 20
18 35
23 26
29 37
8 7
15 42
15 34
10 34
8 33
15 22
12 3
20 29
8 39
42 30
18 6
37 10
41 27
23 2
20 6
20 25
27 18
7 30
32 16
24 32
20 9
41 9
35 7
34 27
18 26
22 32
16 29
41 37
19 7
31 4
30 19
18 1
11 42
23 23
18 39
2 30
34 33
35 39
21 20
17 3
27 9
9 4
42 6
21 21
36 15
17 16
26 32
14 16
31 32
9 36
40 31
38 20
35 42
16 42
26 11
21 18
18 33
5 11
20 16
6 20
9 27
17 40
6 16
37 26
21 9
3 31
5 8
8 40
42 27
41 31
14 19
33 33
17 1
34 37
1 3
32 28
30 36
2 35
5 41
27 6
12 4
21 14
23 25
41 17
27 32
9 2
39 26
22 1
6 40
18 30
34 21
23 17
42 1
1 16
31 11
36 40
19 33
25 11
10 21
26 38
17 24
36 37
14 32
31 38
24 10
20 38
14 34
27 33
5 21
35 11
7 19
3 37
41 3
25 7
41 28
27 5
18 18
15 36
42 5
11 10
33 14
3 27
7 29
31 12
13 10
40 17
1 40
1 21
37 8
38 1
5 15
3 17
16 3
20 30
10 5
16 23
31 13
21 1
42 10
2 27
30 23
1 25
33 28
5 18
9 28
12 8
25 36
16 37
13 9
29 27
36 39
31 6
19 38
10 3
1 42
27 40
4 31
7 38
11 9
41 23
8 34
37 19
9 19
18 41
1 1
9 35
41 12
17 36
11 23
9 23
23 13
11 29
5 42
35 17
38 24
28 24
1 11
8 16
33 10
20 4
29 19
24 35
7 10
23 34
42 7
6 24
33 34
38 4
12 40
29 12
18 7
14 42
16 38
1 24
29 8
13 14
28 38
24 42
42 21
33 8
12 2
2 4
26 7
17 27
36 23
21 31
23 20
20 33
9 32
33 38
8 10
17 9
39 25
6 7
10 16
22 31
16 2
42 9
31 25
7 11
18 34
17 37
24 15
30 22
28 9
7 31
1 12
30 8
13 2
38 22
26 33
33 31
34 13
41 7
2 19
26 39
7 35
24 30
10 31
31 3
2 10
10 4
13 18
26 24
17 39
5 34
28 19
15 27
13 30
15 28
15 14
18 16
42 13
18 22
42 4
9 24
1 20
27 7
23 36
6 12
17 23
17 18
13 39
26 12
7 37
23 5
4 24
37 9
19 25
12 22
38 19
39 42
40 37
9 39
23 7
37 30
20 34
20 21
15 37
28 11
40 3
19 29
18 4
21 28
38 8
11 6
17 30
3 16
39 6
38 5
31 39
29 14
24 31
41 15
28 29
6 5
26 14
11 33
1 22
31 30
12 11
38 7
14 30
29 39
26 5
27 31
39 32
34 5
41 4
6 31
41 26
20 22
10 19
32 20
11 25
5 22
32 34
14 28
4 36
9 5
13 27
25 39
41 24
2 22
24 21
30 42
25 15
5 30
7 14
19 15
16 20
17 38
27 13
41 6
4 5
28 42
41 1
22 25
19 22
29 5
6 14
1 29
8 38
23 35
7 4
8 2
20 13
27 41
12 27
19 28
20 31
25 32
31 31
29 9
3 6
38 39
13 31
9 12
12 7
25 2
24 16
31 41
20 40
20 15
39 29
34 38
8 29
23 19
32 4
12 10
2 29
30 10
39 17
26 27
20 39
3 34
3 24
21 24
27 24
30 2
17 6
36 8
35 9
17 8
7 7
34 3
27 17
3 29
8 24
19 10
16 34
38 35
21 17
24 40
4 37
31 23
8 6
25 34
32 24
26 2
8 13
36 33
15 30
29 31
5 20
21 5
10 10
27 42
10 11
24 25
20 7
1 36
38 21
29 21
32 39
1 2
19 41
29 35
24 27
15 21
6 26
4 22
3 7
38 27
20 5
7 17
25 38
27 34
32 36
2 42
8 27
28 7
14 23
26 8
12 19
32 22
13 36
33 23
18 29
4 18
15 11
6 29
31 35
31 2
32 21
2 31
4 19
5 5
27 12
26 22
10 14
30 11
29 25
40 27
14 13
37 29
27 21
19 12
19 27
3 5
28 33
16 31
10 30
8 26
8 35
5 38
41 33
12 31
17 25
33 32
37 4
38 41
16 14
42 17
35 36
1 26
42 11
26 15
12 30
25 41
17 17
25 9
37 2
42 12
19 3
1 27
42 26
13 16
39 36
36 16
13 23
16 32
15 4
5 17
28 13
12 25
