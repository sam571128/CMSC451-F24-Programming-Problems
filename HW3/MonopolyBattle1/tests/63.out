52 56 66 140 52 56 92 67 56 56 104 56 80 77 52 56 56 67 56 56 75 75 56 56 52 82 56 82 56 147 91 56 153 62 62 66 56 104 56 56 113 56 